tox.ini

```diff
@@ -0,0 +1,26 @@
+[tox]
+envlist =
+    flake8
+    py{27,34,35,36}-dj111-cms{34,35,36}
+    py{34,35,36}-dj20-cms36
+    py{35,36}-dj21-cms36
+
+skip_missing_interpreters=True
+
+[testenv]
+deps =
+    -r{toxinidir}/tests/requirements.txt
+    dj111: Django>=1.11,<2.0
+    dj20: Django>=2.0,<2.1
+    dj21: Django>=2.1,<2.2
+    cms35: django-cms>=3.5,<3.6
+    cms36: https://github.com/divio/django-cms/archive/release/3.6.x.zip
+commands =
+    {envpython} --version
+    {env:COMMAND:coverage} erase
+    {env:COMMAND:coverage} run setup.py test
+    {env:COMMAND:coverage} report
+
+[flake8]
+max-line-length = 120
+exclude = */docs/*,*/migrations/*
```

Comment:
please exclude also .tox/

Reply:
I copied this file from addon to addon, if thats the case, I can mark to update all of them at once. Getting back at a "configuration" addon